ouroboros-consensus/src/Ouroboros/Consensus/Mempool/Impl.hs

```diff
@@ -684,9 +776,7 @@ revalidateTxsFor cfg st lastTicketNo txTickets =
     repeatedly
       (extendVRPrevApplied cfg)
       txTickets
-      (validationResultFromIS is)
```

Comment:
Update the docstring: say that the transactions are *the* transactions in the Mempool and the `TickedLedgerState` corresponds to the chain.